code/modules/antagonists/cult/blood_magic.dm

```diff
@@ -32,20 +32,32 @@
 			B.button.screen_loc = "[screen_loc_X[1]]:[order],[screen_loc_Y[1]]:[screen_loc_Y[2]]"
 			B.button.moved = B.button.screen_loc
 
+/datum/action/innate/cult/blood_magic/proc/get_spell_limit(rune)
+	if(rune)
+		return MAX_BLOODCHARGE
+	return RUNELESS_MAX_BLOODCHARGE
+
+/datum/action/innate/cult/blood_magic/golem/get_spell_limit(rune)
+	if(rune)
+		return MAX_GOLEM_BLOODCHARGE
+	return MAX_GOLEM_BLOODCHARGE_RUNELESS
+
```

Comment:
you can make this a var on the /datum/antagonist/cult and change it on the /datum/antagonist/cult/golem instead of making two procs